autoware_map_msgs/msg/MapProjectorInfo.msg

```diff
@@ -0,0 +1,18 @@
+# Projector type
+string LOCAL = "local"
```

Comment:
I'd prefer it to be camel case as the others, `"Local"` but if it's going to require a lot of changes in other places, it's fine.

Reply:
I think the only code that is relevant is in autoware_map_projection_loader when it is compared against the input from the yaml file.
https://github.com/autowarefoundation/autoware.universe/blob/a9763cbbf929ce380d544563be9c7fd1197f65cc/map/autoware_map_projection_loader/src/map_projection_loader.cpp#L46.

I agree that it's better to keep it consistent with the other strings so I can make it camel case for autoware_msgs. I will modify the node when I port it to Autoware Core from Universe(which I also change from tier4_autoware_msgs to autoware_msgs) to allow both "local" and "Local". I can also post warn messages to the users when they specify "local" and drop the support maybe in the next release.

Reply:
Sounds good :+1: